test/reduce_test.cpp

```diff
@@ -747,32 +747,18 @@ struct reduce_driver : test_driver
     std::vector<std::vector<int>> get_toreduce_dims()
     {
         std::vector<std::vector<int>> tensor_dims = {
-            {0}, {1}, {2}, {3}, {0, 1}, {0, 3}, {0, 2}, {2, 3}, {0, 1, 3}, {1, 2, 3}};
+            {0}, {1}, {2}, {3}, {0, 1}, {0, 3}, {0, 2}, {2, 3}, {0, 1, 3}, {1, 2, 3}, {0, 1, 2, 3}};
 
-        if(std::is_same<T, half_float::half>::value)
-        {
-            tensor_dims.push_back({0, 1, 2, 3});
-
-            return tensor_dims;
-        }
-        else
-            return tensor_dims;
+        return tensor_dims;
     }
 
     reduce_driver()
     {
         add(inLengths, "D", generate_data(get_tensor_lengths()));
         add(toReduceDims, "R", generate_data(get_toreduce_dims()));
-        add(reduceOp, "ReduceOp", generate_data({
-                MIOPEN_REDUCE_TENSOR_ADD, MIOPEN_REDUCE_TENSOR_MIN, MIOPEN_REDUCE_TENSOR_AMAX,
-                    MIOPEN_REDUCE_TENSOR_AVG
-#if MIOPEN_API_VERSION_REDUCE_TENSOR > 1
-                    ,
-                    MIOPEN_REDUCE_TENSOR_NORM1, MIOPEN_REDUCE_TENSOR_NORM2
-#endif
-            }));
+        add(reduceOp, "ReduceOp", generate_data({0, 4, 5, 6, 7}));
```

Comment:
Why symbolic names are not used?

Reply:
I don't know what your means here?   Had I ignored any update made by you?   I just directly used two `#>git revert`.

Reply:
@fronteer You can use symbolic names like `MIOPEN_REDUCE_TENSOR_NORM1` which is more informative.